src/app/assets/languages/en.json

```diff
@@ -121,7 +121,13 @@
       "counter-type": "Counter type",
       "no-counters-status": "No counters yet",
       "no-counters-body": "Click \"Create a counter\" and configure a counter. You can also create counters from the CLI or remote clients.",
-      "no-filtered-counter-body": "No counters match your filter criteria. Click \"Create a counter\" and configure a counter."
+      "no-filtered-counter-body": "No counters match your filter criteria. Click \"Create a counter\" and configure a counter.",
+      "delete-action": "Delete",
+      "add-delta-action": "Add delta",
```

Comment:
Probably we should say somewhere that negative value decrements

Reply:
Should we call this "Set delta" as it was mentioned in [marvel](https://marvelapp.com/prototype/8d66729/screen/69367851)
@dvagnero wdyt?